projects/fal/src/fal/app.py

```diff
@@ -248,6 +248,14 @@ def _to_fal_app_name(name: str) -> str:
     return "-".join(part.lower() for part in PART_FINDER_RE.findall(name))
 
 
+def _print_python_packages() -> None:
+    from pkg_resources import working_set
+
+    print("Python packages installed:")
+    for package in working_set:
+        print(f"{package.key}=={package.version}")
```

Comment:
Can you make this output the following format:

```
[debug] Python packages installed: package==1.2.3, other==3.4.5
```

instead of multiple lines?